app/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -229,4 +229,5 @@
     <string name="no_plugins">无可用插件</string>
     <string name="removed_n_items">已移除 %1$d 个项目</string>
     <string name="whether_reset_switch_preference">是否要将此设置重置为默认值？</string>
+    <string name="clipboard_return_after_paste">粘贴后返回</string>
```

Comment:
As the translation is synced from transifex, please don't modify it manually.

Reply:
I didn't know that. Will remove it then.